modules/raft/src/main/java/org/apache/ignite/internal/raft/storage/impl/DefaultLogStorageFactory.java

```diff
@@ -91,6 +93,9 @@ public class DefaultLogStorageFactory implements LogStorageFactory {
     @SuppressWarnings("ThreadLocalNotStaticFinal")
     private final ThreadLocal<WriteBatch> threadLocalWriteBatch = new ThreadLocal<>();
 
+    /** Rocksdb log flusher, without external access. Used for logs only. */
+    private RocksDbFlusher flusher;
```

Comment:
This looks a bit like an abuse. Is it possible to use a listener instead?

Reply:
Sure, but I'd have to refactor the listener a little bit